lib/ansible/modules/database/postgresql/postgresql_membership.py

```diff
@@ -156,8 +156,8 @@
 def main():
     argument_spec = postgres_common_argument_spec()
     argument_spec.update(
-        groups=dict(type='list', aliases=['group', 'source_role', 'source_roles']),
-        target_roles=dict(type='list', aliases=['target_role', 'user', 'users']),
+        groups=dict(type='list', elements='str', required=True, aliases=['group', 'source_role', 'source_roles']),
+        target_roles=dict(type='list', elements='str', required=True, aliases=['target_role', 'user', 'users']),
```

Comment:
Is there a valid use-case with `groups` or `target_roles` not specified?

Reply:
@felixfontein , no, it isn't. it doesn't make sense, all these parameters are crucial

Reply:
Good, in that case it's fine. I just wanted to make sure that there's no error.